packages/itwinui-react/src/core/Tooltip/Tooltip.tsx

```diff
@@ -251,15 +263,18 @@ export const Tooltip = React.forwardRef((props, forwardRef) => {
             children as JSX.Element,
             tooltip.getReferenceProps({
               ref: childrenRef,
+              ...(ariaStrategy === 'label' && {
+                'aria-labelledby': tooltip.getFloatingProps().id,
+              }),
+              'aria-describedby':
+                ariaStrategy === 'description'
+                  ? tooltip.getFloatingProps().id
+                  : undefined,
```

Comment:
A ternary if-else chain is better suited here

```suggestion
              ...(ariaStrategy === 'label'
                ? {
                    'aria-labelledby': tooltip.getFloatingProps().id,
                  }
                : ariaStrategy === 'description'
                ? {
                    'aria-describedby': tooltip.getFloatingProps().id,
                  }
                : {}),
```

Reply:
i don't think so. This decision is made intentionally - we want to override the `aria-describedby` that comes from floating-ui

Reply:
Ohh, I see, then probably a comment near this code block would help.

Reply:
added